Rocket.Chat/Resources/de.lproj/Localizable.strings

```diff
@@ -4,7 +4,7 @@
 "global.no" = "Nein";
 "global.cancel" = "Abbrechen";
 "global.error" = "Fehler";
-
+chat.message.type.message_snippeted
```

Comment:
This doesn't look right... Can you remove this line and see if the tests pass?

Reply:
Thank you for your reply, i've change the line.